installers/helm/README.md

```diff
@@ -0,0 +1,53 @@
+# Crunchy PostgreSQL Operator
+
+This Helm chart uses the `pgo-deployer` container to install the Crunchy PostgreSQL
+Operator. Helm will setup the ServiceAccount, RBAC, and ConfigMap needed to
+run the container as a Kubernetes Job. Then a job will be created based on `helm`
+`install`, `upgrade`, or `uninstall`. After the job has completed the RBAC will
+be cleaned up.
+
+## Prerequisites
+
+- Helm v3.2.3
+- Kubernetes 1.14+
```

Comment:
Should I update these prereqs? I have developed and tested these changes with Helm v3.2.3 and multiple versions of kube. The oldest version of kube that i have had access to is 1.14 in gke

Reply:
I tested with helm 3.0.0 and didn't run into any issues

Reply:
Helm v3